### PR TITLE
Clarify which snap channel we recommend

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

**Comment by simondeziel** [comment]
@tomponline if this PR goes in, we'll need to update https://discourse.ubuntu.com/t/managing-the-lxd-snap-package/37214. I'm happy to do it unless you want to take care of it.

File: doc/howto/snap.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ When running LXD in a production environment, you must make sure to have a suita
 Snaps come with different channels that define which release of a snap is installed and tracked for updates.
 See [Channels and tracks](https://snapcraft.io/docs/channels) in the snap documentation for detailed information.
 
-Feature releases of LXD are available on the `latest` track.
+Feature releases of LXD are available on the `latest` track as well as numbered tracks like `6` and `6.1`.
```
Comment (tomponline) [inline doc/howto/snap.md:21]: We wont be doing the 6.x tracks anymore

Comment (simondeziel) [reply doc/howto/snap.md]: I welcome that simplification!

Comment (simondeziel) [reply doc/howto/snap.md]: What's the plan with `6.1/*`? Remove it when 6.2 or 6.3 lands in `6/stable`?

File: doc/howto/snap.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ When running LXD in a production environment, you must make sure to have a suita
 Snaps come with different channels that define which release of a snap is installed and tracked for updates.
 See [Channels and tracks](https://snapcraft.io/docs/channels) in the snap documentation for detailed information.
 
-Feature releases of LXD are available on the `latest` track.
+Feature releases of LXD are available on the `latest` track as well as numbered tracks like `6` and `5.21`.
```
Comment (tomponline) [inline doc/howto/snap.md:21]: Lools like the line below this needs to go

File: doc/installing.md [hunk 3]
```diff
@@ -63,6 +64,10 @@ Complete the following steps to install the snap:
 
         sudo snap install lxd --channel=5.0/stable
 
+   For the latest feature release (short support length), use:
```
Comment (tomponline) [inline doc/installing.md:67]: Lets just use 6/stable here to avoid getting people using latest

File: doc/howto/snap.md (hunk 2 above)
Comment (tomponline) [inline doc/howto/snap.md:21]: Lets not mention latest at all to avoid encouraging people using it

**Review by tomponline — APPROVED** [review]

**Pull request merged**